back-end/src/modules/auth/user/UserController.ts

```diff
@@ -0,0 +1,26 @@
+import { BodyParams, ContentType, Controller, Get, PathParams, Post, Returns, ReturnsArray } from '@tsed/common';
+import { MongooseModel } from '@tsed/mongoose';
+import { Inject } from '@tsed/di';
+import { User, UserCreateUpdate } from './User';
+
+
+@Controller('/user')
+export class UserController {
+  @ContentType('application/json')
+  @Inject(User)
+  private userModel: MongooseModel<User>;
+
+  @Post('/register')
+  @Returns(User)
+  async postPayload(@BodyParams() payload: UserCreateUpdate) {
```

Comment:
I would call this register( instead of postPayload because the function gets generated in the front-end